README.md

```diff
@@ -31,6 +31,45 @@ For an upgrade the existing file should be overwritten. On major or structural c
 **Note:**
 [v0.32-alpha](https://github.com/nibi79/synologysurveillancestation/releases/tag/v0.32-alpha) is the last release supporting openHAB prior to 2.4.0. If you want to use newer release please consider to upgrade your openHAB.
 
+## Building the plugin
+
+If you have Java 8 installed simply use the following command to build the plugin
+
+```bash
+mvn clean install
+```
+
+This will create a `target` folder that contains the plugin `jar` file that can be used in your OH2 setup.
+
+### Using docker
+
+The easiest way to build the plugin without setting up your system with all necessary java components is to use the provided `docker` environment.
+In order to build the plugin simply run the following commands. You will find the `jar` files within the same directories as if you would have build the plugin locally.
+
+```bash
+docker-compose run build-plugin
+```
+
+## Developing the plugin
+
+The simplest way to engage in extending the plugin is to use `Eclipse` by following the OH2 developers guide. A good explanation can be found here 
+https://github.com/openhab/openhab2-addons
+
+The steps to setup the plugin in your IDE are the following
+
+1. In the eclipse installer only select `openHAB Development` (And make sure to run JAVA 8) The rest will be taken care of by maven dependencies and you don't need those other projects to develop your binding.
+   1. If you have Eclipse already installed got to `File -> Import... -> Oomph -> GitHub Projects -> openHAB`
+2. Wait till Eclipse finishes startup. No need to do any of: Clean, Build, perform ModelGen launch.
+3. In the `pom.xml` of the demo app. (Next to bnd.runapp) remove the dependency `org.openhab.addons.bom.openhab-addons` And only that dependency.
```

Comment:
3. In the `pom.xml` of the demo app. (Next to bnd.runapp) remove the dependency `org.openhab.addons.bom.openhab-addons` and add following dependency:
```
    <dependency>
      <groupId>org.openhab.addons.bundles</groupId>
      <artifactId>org.openhab.transform.map</artifactId>
      <version>${project.version}</version>
      <scope>runtime</scope>
    </dependency>
```